core/packages/test/scripts/configure-statemine.sh

```diff
@@ -0,0 +1,180 @@
+
+function generate_hex_encoded_call_data() {
+    local type=$1
+    local endpoint=$2
+    local output=$3
+    shift
+    shift
+    shift
+    echo "Input params: $@"
+
+    node ./scripts/helpers/generateHexEncodedCallData.js "$type" "$endpoint" "$output" "$@"
+    local retVal=$?
+
+    if [ $type != "check" ]; then
+        local hex_encoded_data=$(cat $output)
+        echo "Generated hex-encoded bytes to file '$output': $hex_encoded_data"
+    fi
+
+    return $retVal
+}
+
+function send_governance_transact() {
+    local relay_url=$1
+    local relay_chain_seed=$2
+    local para_id=$3
+    local hex_encoded_data=$4
+    local require_weight_at_most_ref_time=$5
+    local require_weight_at_most_proof_size=$6
+    echo "  calling send_governance_transact:"
+    echo "      relay_url: ${relay_url}"
+    echo "      relay_chain_seed: ${relay_chain_seed}"
+    echo "      para_id: ${para_id}"
+    echo "      hex_encoded_data: ${hex_encoded_data}"
+    echo "      require_weight_at_most_ref_time: ${require_weight_at_most_ref_time}"
+    echo "      require_weight_at_most_proof_size: ${require_weight_at_most_proof_size}"
+    echo "      params:"
+
+    local dest=$(jq --null-input \
+                    --arg para_id "$para_id" \
+                    '{ "v3": { "parents": 0, "interior": { "x1": { "parachain": $para_id } } } }')
+
+    local message=$(jq --null-input \
+                       --argjson hex_encoded_data $hex_encoded_data \
+                       --arg require_weight_at_most_ref_time "$require_weight_at_most_ref_time" \
+                       --arg require_weight_at_most_proof_size "$require_weight_at_most_proof_size" \
+                       '
+                       {
+                          "v3": [
+                                  {
+                                    "unpaidexecution": {
+                                        "weight_limit": "unlimited"
+                                    }
+                                  },
+                                  {
+                                    "transact": {
+                                      "origin_kind": "superuser",
+                                      "require_weight_at_most": {
+                                        "ref_time": $require_weight_at_most_ref_time,
+                                        "proof_size": $require_weight_at_most_proof_size,
+                                      },
+                                      "call": {
+                                        "encoded": $hex_encoded_data
+                                      }
+                                    }
+                                  }
+                          ]
+                        }
+                        ')
+
+    echo ""
+    echo "          dest:"
+    echo "${dest}"
+    echo ""
+    echo "          message:"
+    echo "${message}"
+    echo ""
+    echo "--------------------------------------------------"
+
+    npx polkadot-js-api \
```

Comment:
Adapted from cumulus repo scripts. Uses npx to launch node applications instead of expecting it to be in the `PATH` as a global tool.